dds/DCPS/DiscoveryBase.h

```diff
@@ -1635,6 +1635,16 @@ namespace OpenDDS {
         }
 
         DiscoveredParticipantData pdata_;
+        struct LocationUpdate {
+          OpenDDS::DCPS::ParticipantLocation mask_;
```

Comment:
```suggestion
          ParticipantLocation mask_;
```
It's already in the namespace.